framework/test/gtest/FrameworkTest.cpp

```diff
@@ -208,6 +212,56 @@ TEST(FrameworkTest, CustomConfiguration)
 #endif
 }
 
+TEST(FrameworkTest, FrameworkStartsWhenFileNamedDataExistsInTempDir)
+{
+  class ScopedFile
+  {
+  public:
+    ScopedFile(std::string directory, std::string filename) :
+      filePath(std::move(directory) + util::DIR_SEP + std::move(filename))
+    {
+      std::fstream file(filePath, std::fstream::out);
```

Comment:
It is probably good to check the failbit of the stream and if it is bad, fail the test.